cms/models_test.py

```diff
@@ -498,6 +541,26 @@ def test_program_page_for_teams():
     assert not ForTeamsPage.can_create_at(program_page)
 
 
+def test_external_program_page_for_teams():
+    """
+    The ForTeams property should return expected values if associated with an ExternalProgramPage
+    """
+    external_program_page = ExternalProgramPageFactory.create()
+    assert ForTeamsPage.can_create_at(external_program_page)

```

Comment:
Sorry I could not understand it, what is the relation of external programs with teams page ?

Reply:
`For-Teams` page is basically a subpage of the Product Page itself which serves as the parent for all our courseware pages `Courses, External Courses, Programs`(Tests for these pages are added already), hence `Teams` subpage is also supported in External Program page since it also serves as a subpage to the product page, This test just adds the compatibility check for External Programs